ui/src/EmailTemplateList.js

```diff
@@ -276,9 +311,19 @@ class EmailTemplateList extends Component {
         <TableCell component="th" scope="row">
           {JSON.stringify(emailTemplate.metadata)}
         </TableCell>
+        <TableCell component="th" scope="row">
+          {emailTemplate.notifyServiceRef}
+        </TableCell>
       </TableRow>
     ));
 
+    const notifyConfigMenuItems =
+        this.state.notifyServiceRefs.map((supplier) => (
```

Comment:
In general we haven't got comprehensive error handling in the support tool for cases like this, I think in this scenario that API call shouldn't fail, otherwise it indicates a bigger problem with support tool config we should be fixing rather than continuing to use it. For a user facing tool in the future, we'll need a proper error handling strategy that's more graceful.